packages/console/index.ts

```diff
@@ -0,0 +1,9 @@
+// This is a generated file, don't update manually.
+
+import { lazy } from 'react';
+
+import { type Guide } from './types';
+
+const guides: Readonly<Guide[]> = Object.freeze([]);
+
+export default guides;
```

Comment:
What is this file? Seems new to me

Reply:
Auto generated file, @gao-sun could you please take a look?

Reply:
removed